.clj-kondo/config.edn

```diff
@@ -1 +1,3 @@
-{:config-paths ["../resources/clj-kondo.exports/clj-commons/aleph"]}
+{:config-paths ["../resources/clj-kondo.exports/clj-commons/aleph"]
+ :linters      {:unsorted-required-namespaces {:level :warning}}
```

Comment:
This linter isn't enabled by default but I think it's a nice one to have. And since we're likely going to have to touch pretty much all `ns` forms anyway for cleanup ... but feel free to disagree, of course!

Reply:
Agreed. I prefer sorted namespaces, and they're especially cluttered in aleph/manifold.

Sorting should be safe, but in general, my one major concern is that we never take linter warnings as gospel, because there's some really funky stuff in there. Like, namespaces might be required for `extend-type` side effects, but never refer to any vars. And in one case I know of, Zach temporarily switched namespaces in the middle of a file, which plays havoc with modern editors/linters.

Reply:
> Agreed. I prefer sorted namespaces, and they're especially cluttered in aleph/manifold.

Alright, thanks for the feedback!

> Sorting should be safe, but in general, my one major concern is that we never take linter warnings as gospel, because there's some really funky stuff in there.

Sure thing! I will judiciously apply `#_:clj-kondo/ignore` comments :smile: 

> Like, namespaces might be required for extend-type side effects, but never refer to any vars.

`clj-kondo` has a convention for dealing with this - only gotta be careful to apply it where needed :smile: Thanks for the headsup!